yt_dlp/extractor/europarl.py

```diff
@@ -0,0 +1,90 @@
+from .common import InfoExtractor
+from ..utils import parse_iso8601
+
+
+class EuroParlWebstreamIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+                        https?://
+                        (
+                            (multimedia|webstreaming)
+                            \.europarl\.europa\.eu/[^/]+
+                            /(embed/embed\.html\?\bevent=|(?!video)([^/]+)/[\w-]+_)
+                        )
+                        (?P<id>[\w-]+)
+                '''
```

Comment:
So many unused groups...
```suggestion
    _VALID_URL = r'''(?x)
        https?://(?:multimedia|webstreaming)\.europarl\.europa\.eu/[^/#?]+/
        (?:embed/embed\.html\?event=|(?!video)[^/#?]+/[\w-]+_)(?P<id>[\w-]+)
    '''
```